tests/unit/Tag/TagTitleTest.php

```diff
@@ -63,6 +63,8 @@ function () {
                 Tag::setTitle($value);
 
                 expect(Tag::getTitle())->equals("<title>{$value}</title>" . PHP_EOL);
+                
+                expect(Tag::getTitle(false))->equals("{$value}" . PHP_EOL);
```

Comment:
We don't add changes directly to stable branches or `master`. Could you please send this PR to the `3.0.x` branch?